docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "2.1"
+services:
+  fxa-oauth-console:
+    build:
+      context: .
+      dockerfile: Dockerfile-build
+    command: node bin/server.js
+    image: mozilla/fxa-oauth-console
+    ports:
+      - "10137:10137"
```

Comment:
After a lot of searching and then eventually just asking jbuck, I found that network mode host doesn't work on mac. There is currently no way to expose ALL ports on a mac host to the container. Alternatively we can expose each port individually, which would resolve this.